src/main/webapp/app/course/manage/detail/course-detail.component.scss

```diff
@@ -0,0 +1,53 @@
+jhi-secured-image {
+    padding-right: 20px;
+
+    ::ng-deep img {
```

Comment:
Here are some Codacy issues. Are you aware of that?

Reply:
Yes but this was not changed, i just moved the class to another folder. Since css is often wizardry, i decided to leave if like that to not brake anything